src/definitions.ts

```diff
@@ -57,7 +57,7 @@ export abstract class ExportsWalker {
   /** Constructs a new Element walker. */
   constructor(program: Program, includePrivate: bool = false) {
     this.program = program;
-    this.includePrivate;
+    this.includePrivate = includePrivate;
```

Comment:
@dcodeIO I found out that this was not used, I assume that this was the intended use?

Reply:
Whoops, yeah, looks correct now :)